internal/controller/adapters/sveltos/serviceset_controller.go

```diff
@@ -100,7 +100,7 @@ type profileConfig struct {
 type ServiceSetReconciler struct {
 	client.Client
 
-	timeFunc  func() time.Time
+	TimeFunc  func() time.Time
```

Comment:
Could you please elaborate on the reason why this field has changed to the public?

Reply:
For testing purposes only. The tests fail unless this function is set and it's only set in setupwithmanager which can't be called in the tests.

Reply:
The tests are within the package, so i see no reason for this field to have the public scope